charts/super-agent-deployment/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: super-agent-deployment
 description: A Helm chart to install New Relic Super agent on Kubernetes
 
 type: application
-version: 0.0.6-beta
-appVersion: nightly  # TODO: Change this with a proper version of the image.
+version: 0.0.7-beta
+appVersion: "0.6.2"  # TODO: Change this with a proper version of the image.
```

Comment:
the comment is not valid anymore, no?

Reply:
I realized that this is going to fail since that version is expecting the old paths 🤦

Reply:
oh. True.